libraries/AP_RangeFinder/AP_RangeFinder_PulsedLightLRF.cpp

```diff
@@ -140,6 +146,17 @@ static const struct settings_table settings_v2[] = {
     { LL40LS_MEASURE_REG, LL40LS_MSRREG_ACQUIRE },
 };
 
+/*
+  register setup table for V3HP Lidar
+ */
+static const struct settings_table settings_v3hp[] = {
+    { LL40LS_MEASURE_REG, LL40LS_MSRREG_ACQUIRE },
```

Comment:
<img width="475" alt="screen shot 2018-09-08 at 2 36 19 pm" src="https://user-images.githubusercontent.com/15794355/45251688-9019fa80-b374-11e8-86e9-18b5df5f087c.png">

Hi, according to the datasheet, write 0 to this register will perform hard reset, and it may interrupt the I2C lines, so I decided to start a measure step instead. However, I think it is not necessary because we will go to PHASE_MEASURE after init, so this line is not required, and I have removed it from my commit.